packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -8,6 +8,7 @@ export interface IExampleCardProps {
   title: string;
   isOptIn?: boolean;
   code?: string;
+  // tslint:disable-next-line:no-any
   children?: any;
```

Comment:
`React.Children` is a type you could use here. Alternatively, `React.ReactNode[] | React.ReactNode`.